packages/core/docs/guide/getters.md

```diff
@@ -0,0 +1,178 @@
+
+## When should I use them?
+
+**Getters should be used whenever possible to extract the data from composables**, e.g., when you want to get the quantity of the products in the cart in UI components.
+
+## How can I use getters?
+
+The getters accept arguments to get the data. As an example, we will use cart functionalities.
+Most functions in `cartGetters` accept whole `cart` object or individual cart items. This data needs to be extracted from `useCart` composable first.
+As the first example, we will use `getTotalItems` from `cartGetters` to get the total number of items in the cart.
+
+```vue
+<template>
+  <div>
+    <!-- ... -->
+    <span>
+      {{ cartTotalItems }}
+    </span>
+  </div>
+</template>
+
+<script>
+import { computed } from '@vue/composition-api';
+import { useCart, cartGetters } from '{INTEGRATION}';
+import { onSSR } from '@vue-storefront/core';
+
+export default {
+  setup() {
+    const { cart, load } = useCart();
+    const cartTotalItems = computed(() => cartGetters.getTotalItems(cart.value);
+
+    onSSR(async () => {
+      await load();
+    });
+
+    return {
+      cart,
+      cartTotalItems,
+    };
+  }
+}
+</script>
+```
+
+**It's important to use getters as the computed property** to have them always updated:
+
+```vue
+<!-- don't do this: -->
+<script>
+//...
+  export default {
+    setup() {
+      const cartTotalItems = cartGetters.getTotalItems(cart.value);
+      return {
+        cartTotalItems
+      };
+    };
+  }
+</script>
+```
+
+```vue
+<!-- instead do this: -->
+<script>
+//...
+  import { computed } from '@vue/composition-api';
+  export default {
+    setup() {
+      const cartTotalItems = computed(() => cartGetters.getTotalItems(cart.value);
+
+      return {
+        cartTotalItems,
+      };
+    }
+  }
+</script>
+```
+
+You can also use getters in template like so:
+
+```vue
+<!-- or this: -->
+<template>
+  <!-- ... -->
+  <span>
+    {{ cartGetters.getTotalItems(cart) }}
+  </span>
+</template>
+<script>
+//...
+export default {
+  setup() {
+    const { cart, load } = useCart();
+
+    onSSR(async () => {
+      await load();
+    });
+
+    return {
+      cart,
+      cartGetters
+    };
+  }
+};
+</script>
+```
+
+## Available Getters
+
+List of all available getters:
+
+### productGetters
+
+composable: useProduct
+
+[productGetters](#)
+
+### cartGetters
+
+composable: useCart
+
+[cartGetters](#)
+
+### wishlistGetters
+
+composable: useWishlist
+
+[wishlistGetters](#)
+
+### categoryGetters
+
+composable: useCategory
+
+[categoryGetters](#)
+
+### userGetters
+
+composable: useUser
+
+[userGetters](#)
+
+### userShippingGetters
+
+composable: useUserShipping
+
+[userShippingGetters](#)
+
+### userBillingGetters
+
+composable: useUserBilling
+
+[userBillingGetters](#)
+
+### userOrderGetters
+
+composable: useUserOrder
+
+[userOrderGetters](#)
+
+### reviewGetters
+
+composable: useReview
+
+[reviewGetters](#)
+
+### facetsGetters
+
+composable: useFacet
+
+[facetGetters](#)
```

Comment:
```suggestion
```

Let's remove them for now since we don't have API reference yet. I added a note in JIRA.